src/backend/distributed/planner/distributed_planner.c

```diff
@@ -726,19 +722,6 @@ static PlannedStmt *
 InlineCtesAndCreateDistributedPlannedStmt(uint64 planId,
 										  DistributedPlanningContext *planContext)
 {
-	if (!EnableCTEInlining)
```

Comment:
When we deprecate this, even if we suggest `[NOT] MATERIALIZED`, don't we break them? Because they will need to change something. I guess that might be okay but it is still a breaking change I guess